packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md

```diff
@@ -21,7 +23,7 @@ selectAuthenticatorResponse = await _idxClient.SelectChallengeAuthenticatorAsync
     selectAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification` and the Challenge Page should be displayed.
+If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification`, and the Challenge page should be displayed.
```

Comment:
Display is transitive and requires an object to complete the meaning. Appears is intransitive and doesn't require an object to complete the meaning.

```suggestion
If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification`, and the Challenge page should appear.
```

Reply:
fixed